solvers/gtv_mm.py

```diff
@@ -0,0 +1,40 @@
+from benchopt import BaseSolver
+from benchopt import safe_import_context
+
+with safe_import_context() as import_ctx:
+    from benchmark_utils.tv_numba import gtv_mm_tol2
+
+
+class Solver(BaseSolver):
+    """Group TV Denoising with Majoration-Minimisation.
+
+    I. W. Selesnick and P.-Y. Chen, “Total variation denoising with overlapping
+    group sparsity”, 2013
+    doi:10.1109/ICASSP.2013.6638755.
+    """
+
+    name = "Group TV MM"
+    stopping_strategy = "iteration"
+    parameters = {"K": [1, 2, 3, 4, 5]}
```

Comment:
could you give a more informative parameter name and put some comment to document it?